### PR TITLE
Use systemNavigationBarColor to change background color

### PR DESCRIPTION
**Pull request by FaysalSolutions**
- fixes issue #54

### CODE REVIEW

**Review by S1rDan — CHANGES_REQUESTED** [review]
This has still some issues:
- The system navigation bar still doesnt has the correct color and it becomes red / orange after login.
- The status bar within overview is now not readable, if you scroll down.
- Every place, where the app bar casts a shadow, it doesn't transitions neatly into the status bar.

**Comment by FaysalSolutions** [comment]
> This has still some issues:
> 
> * The system navigation bar still doesnt has the correct color and it becomes red / orange after login.


"Resolved in commit [9c514cf](https://github.com/healthcare-it-solutions/picos/pull/133/commits/9c514cfa5016cc956edb7316fadd5ee13b0bfcd5)"

File: lib/app_config.dart [hunk 1]
```diff
@@ -29,8 +30,19 @@ class AppConfig extends StatelessWidget {
   /// AppConfig constructor
   const AppConfig({Key? key}) : super(key: key);
 
+  void _setSystemNavigationBarColor(BuildContext context) {
+    SystemChrome.setSystemUIOverlayStyle(
+      SystemUiOverlayStyle(
+        systemNavigationBarColor:
```
Comment (S1rDan) [inline lib/app_config.dart:36]:
```suggestion
        systemNavigationBarIconBrightness: Brightness.dark,
        systemNavigationBarColor:
```

Made the system buttons more visible.

**Review by S1rDan — CHANGES_REQUESTED** [review]

File: lib/app_config.dart (hunk 1 above)
Comment (FaysalSolutions) [reply lib/app_config.dart:37]: "Resolved in [db5ab9b](https://github.com/healthcare-it-solutions/picos/pull/133/commits/db5ab9b1e94088438b03a1bd060ea2a6bd25640b)"

**Pull request closed without merging**